tool/tsh/tsh.go

```diff
@@ -3923,6 +3928,27 @@ func printStatus(debug bool, p *profileInfo, env map[string]string, isActive boo
 	fmt.Printf("\n")
 }
 
+// outputProfiles displays the provided profile information
+// to the user.
+func outputProfiles(cf *CLIConf, profile *client.ProfileStatus, profiles []*client.ProfileStatus) error {
+	env := getTshEnv()
+	active, others := makeAllProfileInfo(profile, profiles, env)
+
+	format := strings.ToLower(cf.Format)
+	switch format {
+	case teleport.JSON, teleport.YAML:
+		out, err := serializeProfiles(active, others, env, format)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		fmt.Println(out)
+	default:
+		printProfiles(cf.Debug, active, others, env, cf.Verbose)
```

Comment:
`printProfiles` sounds very similar to `outputProfiles`, and calling the wrong one will result in bad things.

What do you think about inlining the contents of `printProfiles` here, so there's only one function for printing profiles?